charts/INSTALL.md

```diff
@@ -0,0 +1,264 @@
+    ```
+### [kubernetes](https://github.com/kubernetes/kubernetes) local cluster
+* You can startup `v1.13.0` k8s local cluster by executing commands below, etcd version 3.2.24 or greater required:
+
+	```
+	cd $HOME
+	git clone https://github.com/kubernetes/kubernetes.git
+	cd $HOME/kubernetes
+	git checkout v1.13.0
+	make
+	echo alias kubectl='$HOME/kubernetes/cluster/kubectl.sh' >> /etc/profile
+	ALLOW_PRIVILEGED=true FEATURE_GATES=CSIPersistentVolume=true,MountPropagation=true,VolumeSnapshotDataSource=true,KubeletPluginsWatcher=true RUNTIME_CONFIG="storage.k8s.io/v1alpha1=true" LOG_LEVEL=5 hack/local-up-cluster.sh
+	```
+	
+### Helm Installation ###
+* This will fetch the latest version of helm :
+
+	```
+	curl https://raw.githubusercontent.com/helm/helm/master/scripts/get >get_helm.sh
+	chmod 700 get_helm.sh
+	./get_helm.sh
+	helm init
+    ```
+### Keystone installation and configuration ###
+
+* Keystone Installation
+	```
+	git clone https://github.com/opensds/opensds-installer.git
+	cd opensds-installer/ansible
+	chmod +x ./install_ansible.sh && ./install_ansible.sh
+	cd script
+	./keystone.sh install 
+	./keystone.sh config hotpot
+    ```
+### Ceph Installation
+* Here we are creating only 1 ceph-osd node and Ceph luminous version ( For more details please refer [link](http://docs.ceph.com/ceph-ansible/master/)  )
+* Clone Ceph repository
+	```
+	cd $HOME/kubernetes
+	git clone -b stable-3.0 https://github.com/ceph/ceph-ansible.git
+	
+* Configure ```ceph-ansible/group_vars/all.yml``` by appending below portion:
+
+    ```
+    ceph_origin: repository
+    ceph_repository: community
+    ceph_stable_release: luminous # Choose luminous as default version
+    public_network: "192.168.3.0/24" # Run 'ip -4 address' to check the ip address
+    cluster_network: "{{ public_network }}"
+    monitor_interface: eth1 # Change to the network interface on the target machine
+    devices: # For ceph devices, append ONE or MULTIPLE devices like the example below:
+      - '/dev/sda' # Ensure this device exists and available if ceph is chosen
+      #- '/dev/sdb'  # Ensure this device exists and available if ceph is chosen
+    osd_scenario: collocated
+    ```
+
+* You can manually create ceph.hosts file OR simply copy it from opensds-installer repo as follow: 
+	```
+	cd $HOME/kubernetes
+	cp opensds-installer/ansible/group_vars/ceph/ceph.hosts ceph-ansible/
+	# Install ceph cluster
+	cd $HOME/kubernetes/ceph-ansible
+	ansible all -m ping -i ceph.hosts
+	ansible-playbook site.yml -i ceph.hosts | tee /var/log/ceph_ansible.log
+	# Create pools if not available
+	ceph osd crush tunables hammer
+	grep -q "^rbd default features" /etc/ceph/ceph.conf || sed -i '/\[global\]/arbd default features = 1' /etc/ceph/ceph.conf
+	ceph osd pool create rbd 100 && ceph osd pool set rbd size 1   
+
+### OpenSDS helm chart installation
+
+## Pre-configuration
+* Before you start, some configurations are required:
+
+    ```
```

Comment:
The configure file is out of date, please update it to latest version.

Reply:
@leonwanghui  I have made changes for config file, Thanks.